qgepplugin/swmm/swmm_summary.qgs

```diff
@@ -0,0 +1,8075 @@
+<!DOCTYPE qgis PUBLIC 'http://mrcc.com/qgis.dtd' 'SYSTEM'>
```

Comment:
What is this file for? I don't see any reference elsewhere in the code.

Reply:
These two qgis projects are explained here: https://qgep.github.io/docs/qgep_swmm/Mapping.html#results-are-imported-in-qgep